db/re/skill_db.yml

```diff
@@ -5948,7 +5947,7 @@ Body:
     Hit: Single
     HitCount: 1
     Element: Dark
-    Duration2: 30000
+    Duration2: 18000
```

Comment:
Can you test NPC_CURSEATTACK please? My last tests showed the duration was 30s unlike all other curse skills which were 20s.

Based on that test the value here should be 28000 (or maybe 30000?).

Also, since it then can't be used as default duration, you need to change status.yml as well. Use NPC_WIDECURSE for the default duration instead. We already did the same for freeze which references NPC_WIDEFREEZE, so it would be consistent.